FHEM/lib/SD_Protocols.pm

```diff
@@ -1743,6 +1743,41 @@ sub _xFSK_methods_behind_here {
   # only for functionslist - no function!
 }
 
+=item ConvBresser_5in1()
+
+This function checks number/count of set bits within bytes 14-25 and inverted data of 13 byte further.
+Delete inverted data (nibble 1-27)and reduce message length (nibble 53).
+
+Input:  $hexData
+Output: $hexData
+        scalar converted message on success 
+        or array (1,"Error message")
+
+=cut
+
+sub ConvBresser_5in1 {
+  my $self    = shift // carp 'Not called within an object';
+  my $hexData = shift // croak 'Error: called without $hexdata as input';
+  my $d2;
+  my $bit;
+  my $bitsumRef;
+  my $bitadd = 0;
+  for (my $i = 0; $i < 13; $i++) {
+    $d2 = hex(substr($hexData,($i+13)*2,2));
+    return ( 1, qq[ConvBresser_5in1, inverted data at pos $i] ) if ((hex(substr($hexData,$i*2,2)) ^ $d2) != 255);
```

Comment:
Für diesen Fall bräuchte ich einen passenden Testdatensatz als input.

Reply:
Dazu bräuchtest du nur irgendein Nibble oder Byte in einer "guten" RAWMSG zu ändern.

Reply:
Okay der war einfach. Hat so geklappt